ansible/roles/test/files/ptftests/py3/IP_decap_test.py

```diff
@@ -74,6 +74,7 @@ class DecapPacketTest(BaseTest):
     DEFAULT_INNER2_V6_PKT_DST_IP = '3::3'
 
     # Allowed DSCP and TTL values
+    DSCP_EXCLUDE = {2, 6}
```

Comment:
We can skip this DSCP only when
1. The topology is T1
2. tunnel_qos_remap is enabled. Can be checked with `is_tunnel_qos_remap_enabled`